tests/Http/Controllers/Api/Entries/View/External/ExportRoutes/ApiRateLimits/RateLimitsMediaExportLocalTest.php

```
@@ -0,0 +1,285 @@
+            }
+
+            $this->cleanUp();
+        } catch (GuzzleException $e) {
+            // On exception, log and clean up
+            $this->cleanUp();
+            $this->logTestError($e, []);
+            return false;
+        }
+
+        return true;
+    }
```

PHPMD (2.15.0)

Remove error control operator '@' on line 183. (Clean Code Rules)

(ErrorControlOperator)